contracts/governance/Vesting/VestingRegistryProxy.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity ^0.5.17;
+
+import "./VestingRegistryStorage.sol";
+import "../../proxy/UpgradableProxy.sol";
+
+/**
+ * @title Vesting Registry Proxy contract.
+ * @dev Vesting Registry contract should be upgradable, use UpgradableProxy.
+ * VestingRegistryStorage is deployed with the upgradable functionality
+ * by using this contract instead, that inherits from UpgradableProxy
+ * the possibility of being enhanced and re-deployed.
+ * */
+contract VestingRegistryProxy is VestingRegistryStorage, UpgradableProxy {
```

Comment:
`UpgradableProxy` contract is similar to EIP-1967 which means that it would be better if the proxy does not know about the logic of `VestingRegistryLogic` and its storage

Reply:
We have used the same approach in Staking and it works well. I am curious to understand your suggestion here.